_includes/tutorials/produce-consume-lang/scala/markup/dev/4_0-helper-package.adoc

```diff
@@ -0,0 +1,10 @@
+Topic creation and avro schema declaration are often part of an external process. For the sake of clarity in this
+tutorial, we won't include these steps as part of the main application, but isolate theme in a dedicated package.
+
```

Comment:
This is well authored!  Very clear and concise explanation of the purpose of the sbt packages and how these functions are usually performed outside the application.